src/openapi/paths/funder.json

```diff
@@ -0,0 +1,91 @@
+{
+	"get": {
+		"operationId": "getFunderById",
+		"summary": "Gets a specific funder.",
+		"tags": ["Funders"],
+		"security": [
+			{
+				"auth": []
+			}
+		],
+		"parameters": [
+			{
+				"name": "funderShortCode",
+				"description": "The short code of a funder.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "string"
+				}
+			}
+		],
+		"responses": {
+			"200": {
+				"description": "The funder.",
+				"content": {
+					"application/json": {
+						"schema": {
+							"$ref": "../components/schemas/Funder.json"
+						}
+					}
+				}
+			},
+			"404": {
+				"$ref": "../components/responses/NotFound.json"
+			}
+		}
+	},
+	"put": {
+		"operationId": "createOrUpdateFunder",
+		"summary": "Creates or updates a new funder.",
+		"tags": ["Funders"],
+		"security": [
+			{
+				"auth": []
+			}
+		],
+		"parameters": [
+			{
+				"name": "funderShortCode",
+				"description": "The short code of a funder.",
```

Comment:
Totally unrelated to this PR, I noticed that we should not provide a legacy short code (`pdc`) in the example of a funder short code. It would be better to provide an invalid example to highlight that we expect the caller to create one if it does not exist.

Reply:
Created https://github.com/PhilanthropyDataCommons/service/issues/1602.